Update Gitignore

austinsellers opened the pull request:
- Remove unnecessary files 
- Update .gitignore

austinsellers committed:
Update Gitignore

Remove unnecessary files and update .gitignore

jgugli3 reviewed (commented):
Thats a lot of changes. But I cant do anything about it on these computers

austinsellers committed:
Keep Config Folder

Update .gitignore to keep config files for project because previous commit
did not work when this was removed.
